extensions/vscode/CONTRIBUTING.md

```diff
@@ -50,17 +50,46 @@ just test
 
 Utilize VSCode's built-in debugger to troubleshoot and diagnose issues.
 
-Launch the debugger by pressing `F5` or go to `Run > Start Debugging`.
+Switch to the debugger extension view within VSCode and select one of the two
+debug configurations (specified within `extensions/vscode/.vscode/launch.json`):
+
+- `Debug with internal API agent`
```

Comment:
I'd suggest simplifying the name to "Run Extension" since it implies a bundled agent already. Nitpicky though, feel free to ignore.

Reply:
no problem.